### PR TITLE
Patch/ticket in nulls

### PR DESCRIPTION
**Pull request by sternhenri**
Three big changes:
- Diagrams added
- Precision of what to do if multiple blocks in a TipSet have losing tickets
- spec'd ticket comparison steps.

### CODE REVIEW

File: expected-consensus.md [hunk 1]
```diff
@@ -352,7 +433,7 @@ potentially long chain scans would be required to compute a given block's weight
 
 ### Selecting between TipSets with equal weight
 
-When selecting between TipSets of equal weight, a miner chooses the one with the smallest min ticket (by bytewise comparison).
+When selecting between TipSets of equal weight, a miner chooses the one with the smallest min ticket.
```
Comment (ZenGround0) [inline expected-consensus.md]: My reading of the above discussion is that tickets are comparable against tickets but I don't know how to compare ticket arrays with each other.  Comparing ticket arrays is the thing we really want for 1) tie breaking, 2) consistent message application order.  It is the thing we should include in the spec.

Right now I'm imagining we just take the first ticket of the array and compare.

Comment (ZenGround0) [reply expected-consensus.md]: Reading above I'm actually guessing its the final ticket, not the first.  But let's make sure to make that clear.

Comment (sternhenri) [reply expected-consensus.md]: Understood. Yes! Thanks

**Comment by sternhenri** [comment]
outdated

**Pull request closed without merging**